README.md

```diff
@@ -63,4 +63,14 @@ implementation for review after the RFC has been accepted.
 When a pull request has implemented an RFC, the RFC should be updated with a link
 to the PR implementing it.
 
+## Merging an RFC
+
+An RFC may be merged by a TSC member provided that:
+
+1. Every TSC member who wishes to comment on the RFC has done so.
+1. All outstanding concerns and comments have been addressed.
+1. Consensus has been reached on merging the RFC (no TSC member dissents to merging). This may be done on the RFC pull request by TSC members marking the pull request as approved.
```

Comment:
To clarify, does this also require that a quorum of TSC members have approved the PR (in addition to having no dissenters)?

Reply:
Yes, I'd envision this as the same process we follow for core changes.